x-pack/plugin/fleet/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/fleet/20_wait_for_checkpoints.yml

```diff
@@ -105,6 +105,7 @@ setup:
         index: "test-after-refresh"
         allow_partial_search_results: false
         wait_for_checkpoints: 2
+        wait_for_checkpoints_timeout: 1m
```

Comment:
A timetous >30sec is needed in order to bypass the condition set by https://github.com/elastic/elasticsearch/pull/96145 and make the "wait for unissued seqno" error to appear. Here and in the below cases.